Bookish/Controllers/BookController.cs

```diff
@@ -0,0 +1,88 @@
+using System.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using Bookish.Models;
+using Microsoft.AspNetCore.Authorization.Infrastructure;
+
+namespace Bookish.Controllers;
+
+public class BookController : Controller
+{
+    public IActionResult Catalogue()
+    {
+        using var context = new LibraryContext();
+        var books = context.Books.ToList();
+        return View(books);
+    }
+    
+    [HttpGet]
+    public IActionResult AddBook()
+    {
+        return View();
+    }
+    
+    [HttpPost]
+    public ActionResult AddBook(BookModel newBook)
+    {
+        EditBooks.InsertBooks(newBook);
+        return View(newBook);
+    }
+    [HttpGet]
+    public ActionResult EditBook(int id)
+    {
+        using var context = new LibraryContext();

```

Comment:
Same here, would be nice to have an EditBooks.GetBookById() method, or similar

Reply:
Created a method in EditBooks to do this (GetBookFromId) - I've also created equivalent database-interaction methods in EditMembers.